### DISCUSSION TITLE
Expo-blur usage

### DISCUSSION THREAD

**Discussion by Pyroboomka** [enesozturk/react-native-hold-menu#14] [Ideas]
In bare react native project, this library requires an unimodules integration to start up, which is not listed anywhere.
Maybe replace expo-blur with https://github.com/Kureev/react-native-blur to not force people in all-that-unimodules-setup-shenanigans.

**Comment by enesozturk** [comment]
Hi @Pyroboomka, thanks for advice. Actually several reasons to use unimodules. Before `expo-blur` I was using react-native-blur. I realized that `expo-blur` was more performant when create animated component from it. Then when we thing about haptic feedbacks, `expo-haptic` was better choice than `react-native-haptic-feedback`. Otherwise Expo users will not able to use haptic.

At the any optional package usage can be used for blur and haptic of course. I will consider it in further releases.

**Reply by enesozturk** [reply]
Hi, it is worth mentioning for further visitors that when use react-native-unimodules, you can exclude modules that you don't want to include. Check https://docs.expo.io/bare/installing-unimodules/
